doc/sphinx/addendum/type-classes.rst

```diff
@@ -527,7 +527,9 @@ Options
 
 .. flag:: Typeclasses Debug
 
-   Controls whether typeclass resolution steps are shown during search.  Setting this flag
+.. flag:: Debug Typeclasses
+
+   Two names for the same option. Controls whether typeclass resolution steps are shown during search.  Setting this flag
```

Comment:
One of these two names is marked as deprecated: remove it instead of documenting it.

Reply:
Neither one is currenlty marked as deprecated.  I will remove "Debug Typeclasses" from the rst and mark it as deprecated.